Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -58,11 +58,12 @@ async function main(): Promise<void> {
         
         // Getting NuGet
         tl.debug('Getting NuGet');
-        let versionSpec = tl.getInput('versionSpec', true);
-        let checkLatest = tl.getBoolInput('checkLatest', false);
         let nuGetPath: string = undefined;
         try {
-            nuGetPath = await nuGetGetter.getNuGet(versionSpec, checkLatest);
+            nuGetPath = process.env[nuGetGetter.NUGET_EXE_TOOL_PATH_ENV_VAR];
```

Comment:
This is ok, but IMO the nuGetGetter should not leak this abstraction -- the nuGetInstaller should just call a method like nuGetGetter.GetNuGet(fallbackDefaultVersionToUseIfNoneHasBeenSet) and let nuGetGetter handle all the logic determining whether a NuGetVersion has already been set or to use the supplied fallback version.